sdk/search/azure-search-documents/tests/async_tests/test_index_live_async.py

```diff
@@ -239,7 +239,8 @@ class SearchIndexClientTestAsync(AzureMgmtTestCase):
             assert set(x.status_code for x in results) == {201}
 
             # There can be some lag before a document is searchable
-            time.sleep(3)
+            if self.is_live:
+                time.sleep(3)
```

Comment:
I know times can vary depending on what you're waiting for. Maybe define this as a global at the file scope so it can be changed if necessary?